### DISCUSSION TITLE
Issues using headless mode and CI

### DISCUSSION THREAD

**Discussion by Kulezic101** [ortoniKC/ortoni-report#37] [General]
Hi,
We use Jenkins automation to run Playwright tests and we have **open: process.env.CI ? 'never' : 'always',**
configuration in the playwright.config.ts file.  However, this setting does not seem to play well with "Show browser" option of "Playwright Test for VS Code" by Microsoft.

Regards,
Igor

**Comment by ortoniKC** [comment]
Does it work for the playwright html report?

**Comment by Kulezic101** [comment]
Sorry for the late reply.  Yes it does work with html report.   In fact when running locally we now use HTML report and ortoni-report on CI.
